railties/lib/rails/generators/generated_attribute.rb

```diff
@@ -75,7 +75,7 @@ def field_type
                         when :date                 then :date_select
                         when :text                 then :text_area
                         when :boolean              then :check_box
-          else
+                        else
                           :text_field
         end
```

Comment:
Hrmm.. seeing the `else` aligned with the `when` is nice, but this still sounds off. Is there something else we want to enable as well?

TBH, I'm not sure how this should be arranged at all -- my personal taste is more for `keyword` alignment. @rafaelfranca should the `when` be aligned with the variable, like an `else` on an `if` would? :confused:

Reply:
Yeah. I think that should be the case

Reply:
Thanks for pointing that out. I was insufficient self code review. This is indeed strange.

I investigated the alignment of `when` using master branch of RuboCop.
I think the current RuboCop does not have ability to align `when` with the variable (RuboCop's default is `end`  alignment with `keyword`, so there may be weak points in alternative setting `variable`.) I will continue to investigate so that will be able to align `when` with the variable in future upgrades of RuboCop.

I'd like to make improvements to `if ... else ... end` the main objective of this PR.
For improvement on align `when` with the variable, I'd like to open another PR after the upgrade RuboCop in the future. 

What do you think about it?

Reply:
Yeah, if you don't know of some other setting we're just missing, I agree we should leave that for another PR; this misalignment is already there, so the change makes it no worse -- and the others are much better :+1: